packages/edit-post/src/hooks/use-navigate-to-entity-record.js

```diff
@@ -52,7 +56,23 @@ export default function useNavigateToEntityRecord(
 	const { setRenderingMode } = useDispatch( editorStore );
 
 	const onNavigateToEntityRecord = useCallback(
-		( params ) => {
+		async ( params ) => {
+			try {
+				await registry
+					.resolveSelect( coreStore )
+					.getPostType( params.postType );
```

Comment:
@getdave, can you explain why we need to pre-resolve this selector?

Reply:
Yes it would help. Let me see if I can dig this up. We should include information inline in a comment.

Reply:
I think this is probably fixing a symptom rather than fixing the root issue where we may be using an undefined postType object or something.